src/components/TrainerProfile/index.tsx

```diff
@@ -0,0 +1,9 @@
+import * as S from './styles'
```

Comment:
Se esse componente so for utilizado dentro do Team, e nao tiver nenhum comportamento espeficio, acho que esse componente e meio inutil tambem ^^

Reply:
Hehe, nesse caso eu tô planejando colocar um menu nele, até fiz algumas alterações no layout geral... Mas isso é mais pra frente, por isso que preferi deixar como componente isolado.